packages/core/test/test-invoke.js

```diff
@@ -359,7 +359,7 @@ QUnit.module('Invoke', hooks => {
 
   QUnit.test('Invoking a machine that immediately finishes', async assert => {
     assert.expect(3);
-    const expectations = [ 'two', 'nestedTwo', 'three' ];
+    const expectations = [ 'nestedTwo', 'three', 'three' ];
```

Comment:
Agreed that this is the expected behaviour. For the next breaking (major) release, I'd say that it makes a lot of sense to have `const expectations = [ 'two.nestedTwo', 'three', 'three' ];` because it aligns with the recently added `.matches()` API.

Reply:
Should we collect these somewhere?